internal/test/data/dummyCa.pem

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
If we want to add another test data, we have to enter it in this directory (internal / test / data).
If the number of files becomes large, it will be complicated, so it is better to put data in each directory.
eg)
```
internal/test/data/tls/dummyCa.pem
internal/test/data/config/config.yaml
internal/test/data/config/invalid_config.yaml
...etc
```